editor/editor_settings_dialog.cpp

```diff
@@ -175,10 +175,17 @@ void EditorSettingsDialog::shortcut_input(const Ref<InputEvent> &p_event) {
 			handled = true;
 		}
 
-		if (k->get_keycode_with_modifiers() == (KeyModifierMask::CMD | Key::F)) {
+#ifdef MACOS_ENABLED
+		if (k->get_keycode_with_modifiers() == (KeyModifierMask::META | Key::F)) {
 			_focus_current_search_box();
 			handled = true;
 		}
+#else
+		if (k->get_keycode_with_modifiers() == (KeyModifierMask::CTRL | Key::F)) {
+			_focus_current_search_box();
+			handled = true;
+		}
+#endif
```

Comment:
Couldn't this use `KeyModifierMask::CMD_OR_CTRL` instead of the `#ifdef`?

Reply:
Not directly, since a real event won't have it set. But it should be possible to rewrite it as comparison of a real event and new event with a cmd_or_ctrl set, instead of directly comparing key masks.

Reply:
Updated to use `is_match` without `ifdef`s.